### PR TITLE
Home work 07

### PR DESCRIPTION
**Pull request by userhost1**
Hi. Check please:
Transport, 
part2task27СreditCards, 
part2task26Appliances.

### CODE REVIEW

File: src/homework07/Transport/CivilTransport.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                ", wingspan=" + wingspan +
+                ", minDistanceForTakeoff=" + minDistanceForTakeoff +
+                ", horsePower=" + horsePower +
+                ", maxSpeed=" + maxSpeed +
+                ", weightKG=" + weightKG +
+                ", brand='" + brand + '\'' +
+                ", kWPower=" + getkWPower() + '}';
+    }
+
+    public double getkWPower() {
```
Comment (TheFallen058) [inline src/homework07/Transport/CivilTransport.java]: getKWPower will be correct naming due to camelcase

**Review by TheFallen058 — APPROVED** [review]
well done

File: src/homework07/Transport/CivilTransport.java (hunk 1 above)
Comment (userhost1) [reply src/homework07/Transport/CivilTransport.java]: thanks

**Comment by userhost1** [comment]
fixed

**Pull request merged**